lib/usr/app/jcfg/jcfg_umem.c

```diff
@@ -181,3 +187,12 @@ _decode_umems(struct json_object *obj, int flags, struct json_object *parent __c
 
     return ret ? ret : JSON_C_VISIT_RETURN_SKIP;
 }
+
+void
+jcfg__umem_free(jcfg_hdr_t *hdr)
+{
+    jcfg_umem_t *umem = (jcfg_umem_t *)hdr;
+
+    free(umem->rinfo);
```

Comment:
Can you check if umem is null here?

Reply:
umem can not be NULL here as it is only called from one spot.